README.md

```diff
@@ -14,22 +14,20 @@ Example:
     function* handle(msg) { ... };
     function* validate(msg) { ... };
 
-    const worker = workerlib.createWorker(
-        handle,
-        {
-          workerName: 'my worker',
-          amqpUrl: 'amqp://guest:guest@localhost:5672',
-          exchangeName: 'bus',
-          queueName: 'test.test_watcher',
-          routingKey: 'test.something_happened'
-        },
-        {
-            validator: validate,
-            channelPrefetch: 50,
-            taskTimeout: 30000,
-            processExitTimeout: 3000
-        }
-    );
+    const worker = workerlib.createWorker([{
+      handle: handle,
+      validate: validate,
+      routingKey: 'test.something_happened'
+    }], {
+      workerName: 'my worker',
+      amqpUrl: 'amqp://guest:guest@localhost:5672',
+      exchangeName: 'bus',
+      queueName: 'test.test_watcher'
+    }, {
+      channelPrefetch: 50,
+      taskTimeout: 30000,
+      processExitTimeout: 3000
+    });
```

Comment:
Problem: this is an example of how to use `createWorkers` with the `createWorker` name